### ISSUE TITLE
Computing with Google Collab - error to load specific environment

### ISSUE THREAD

**Issue by pa-laharotte** [licit-lab/ITSProjects#1]
* Project No: 1 (V2X)

### General review

The following error occurs when we try to compute it directly on Google Collab:
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

It seems that it is not pointing out the right folder, but I don't know how to find the right 

```
!pip install -r requirements.txt

ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
```

As an alternative, I tried to compute with the following code, but it remains unsuccessful

```
!pip install -r https://github.com/research-licit/ITSProjects/blob/main/requirements.txt

ERROR: Invalid requirement: '<!DOCTYPE html>' (from line 7 of https://github.com/research-licit/ITSProjects/blob/main/requirements.txt)
```

Visited websites:
https://github.com/ml5js/ml5-library/issues/954
https://github.com/projectmesa/mesa/issues/315

**Comment by pa-laharotte** [comment]
The error disappears when compiling directly
```
# Try this on google collab only
#!pip install -r requirements.txt
```

but I am not sure the reqirements.txt file is taken into account.

Indeed it does not recognize the `connectv2x` package
```
from connectv2x.traffic import FundamentalDiagram
```

A further question is: Why does the requirements.txt & environment.yaml point on research-licit/connectv2x/ package and not on research-licit/ITSTools/connectv2X/ folder ?

**Comment by aladinoster** [comment]
His thas been fixed in 1d97620949ac98a378a2d1deface70d1cea0d098

> A further question is: Why does the requirements.txt & environment.yaml point on research-licit/connectv2x/ package and not on research-licit/ITSTools/connectv2X/ folder ?

The used to point there because they were standard stable repositories it is not the case anymore
